supportedBackends/htcondor/src/test/scala/cromwell/backend/impl/htcondor/HtCondorRuntimeAttributesSpec.scala

```diff
@@ -237,25 +243,48 @@ class HtCondorRuntimeAttributesSpec extends WordSpecLike with Matchers {
       val shouldBeIgnored = workflowOptionsWithDefaultRA(Map())
       assertHtCondorRuntimeAttributesSuccessfulCreation(runtimeAttributes, shouldBeIgnored, expectedRuntimeAttributes)
     }
+
+    "return an instance of itself when tries to validate a valid native specs entry" in {
+      val expectedRuntimeAttributes = staticDefaults.copy(nativeSpecs = Option(Array("spec1", "spec2")))
+      val runtimeAttributes = createRuntimeAttributes(HelloWorld, """runtime { nativeSpecs: ["spec1", "spec2"] }""").head
+      assertHtCondorRuntimeAttributesSuccessfulCreation(runtimeAttributes, emptyWorkflowOptions, expectedRuntimeAttributes)
+    }
+
+    "throw an exception when tries to validate an invalid native specs entry" in {
+      val runtimeAttributes = createRuntimeAttributes(HelloWorld, """runtime { nativeSpecs: [1, 2] }""").head
+      assertHtCondorRuntimeAttributesFailedCreation(runtimeAttributes, "Expecting nativeSpecs runtime attribute to be an Array of Strings.")
+    }
   }
 
-  private def assertHtCondorRuntimeAttributesSuccessfulCreation(runtimeAttributes: Map[String, WdlValue], workflowOptions: WorkflowOptions, expectedRuntimeAttributes: HtCondorRuntimeAttributes): Unit = {
+  private def assertHtCondorRuntimeAttributesSuccessfulCreation(runtimeAttributes: Map[String, WdlValue],
+                                                                workflowOptions: WorkflowOptions,
+                                                                expectedRuntimeAttributes: HtCondorRuntimeAttributes) = {
     try {
-      assert(HtCondorRuntimeAttributes(runtimeAttributes, workflowOptions) == expectedRuntimeAttributes)
+      val actualRuntimeAttr = HtCondorRuntimeAttributes(runtimeAttributes, workflowOptions)
+      assert(actualRuntimeAttr.cpu == expectedRuntimeAttributes.cpu)
+      assert(actualRuntimeAttr.disk == expectedRuntimeAttributes.disk)
+      assert(actualRuntimeAttr.memory == expectedRuntimeAttributes.memory)
+      assert(actualRuntimeAttr.continueOnReturnCode == expectedRuntimeAttributes.continueOnReturnCode)
+      assert(actualRuntimeAttr.failOnStderr == expectedRuntimeAttributes.failOnStderr)
+      assert(actualRuntimeAttr.dockerWorkingDir == expectedRuntimeAttributes.dockerWorkingDir)
+      assert(actualRuntimeAttr.dockerImage == expectedRuntimeAttributes.dockerImage)
+      assert(actualRuntimeAttr.dockerOutputDir == expectedRuntimeAttributes.dockerOutputDir)
+      expectedRuntimeAttributes.nativeSpecs match {
+        case Some(ns) => assert(ns.deep == expectedRuntimeAttributes.nativeSpecs.get.deep)
+        case None => assert(expectedRuntimeAttributes.nativeSpecs.isEmpty)
+      }
     } catch {
       case ex: RuntimeException => fail(s"Exception was not expected but received: ${ex.getMessage}")
     }
-    ()
```

Comment:
If Travis is happy, it looks like it's ok. The preceeding block probably already returned Unit (we only get the compiler error when something is returning a real value from a `=> Unit` function)